Source/ASDisplayNode+Layout.mm

```diff
@@ -287,8 +287,9 @@ - (void)displayNodeDidInvalidateSizeNewSize:(CGSize)size
   }
 }
 
-- (void)_locked_measureNodeWithBoundsIfNecessary:(CGRect)bounds
+- (void)_u_measureNodeWithBoundsIfNecessary:(CGRect)bounds
 {
```

Comment:
Same, let's call `ASDisplayNodeAssertLockUnownedByCurrentThread()` before acquiring the lock.